dll/cpl/desk/lang/cs-CZ.rc

```diff
@@ -147,15 +147,16 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Obecné"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Obrazovka", -1, 7, 6, 237, 56, WS_GROUP
-    LTEXT "Velikost písma", -1, 14, 20, 222, 9
-    COMBOBOX IDC_FONTSIZE_COMBO, 14, 30, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
-    GROUPBOX "Kompatibilita", -1, 7, 67, 237, 97
-    LTEXT "Některé programy nefungují správně, pokud není počítač restartován po změně nastavení barev.\r\n\nPo změně nastavení barev:", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&Restartovat počítač před aplikováním nového nastavení barev.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "Aplikovat nové nastavení barev bez restartování.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "Zeptat se před aplikováním nového nastavení barev.", IDC_ASKME_RB, 14, 146, 228, 10
+    GROUPBOX "Obrazovka:", -1, 7, 6, 237, 105, WS_GROUP
+    LTEXT  "If the screen resolution makes items too small to view comfortably, you can increase the ppp value to compensate. To change the font size only, click ""Cancel"" and go to the ""Appearance"" tab.", -1, 14, 20, 220, 45
+    LTEXT "Velikost písma:", -1, 14, 65, 222, 9
+    COMBOBOX IDC_FONTSIZE_COMBO, 14, 75, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 91, 223, 12
+    GROUPBOX "Kompatibilita", -1, 7, 115, 237, 100
+    LTEXT "Some programs may not function properly if you do not restart your computer after changing the display settings.\n\nAfter changing the display settings:", -1, 12, 130, 228, 34
+    AUTORADIOBUTTON "& Restart the computer before applying the new settings", IDC_RESTART_RB, 14, 165, 228, 10
+    AUTORADIOBUTTON "& Apply new settings without restarting", IDC_WITHOUTREBOOT_RB, 14, 180, 228, 10
+    AUTORADIOBUTTON "& Ask before applying new display settings", IDC_ASKME_RB, 14, 195, 228, 10
```

Comment:
Why did you edit this file?

Reply:
To add :
     LTEXT "If the screen resolution makes items too small to view comfortably, you can increase the ppp value to compensate. To change the font size only, click" "Cancel" "and go to the" "Appearance" "tab.", - 1, 14, 20, 220, 45
and I copied the entire dialog box setting because I had to readjust the position of the objects to fit the text.
As Hbelusca asked I added this definition to all languages